LGB.py

```diff
@@ -248,3 +248,7 @@ def eval_f(y_true,y_pred):
 # print(test['preds'].mean())
 test[['id','preds']].to_csv('submission/lgb_pred_test.csv', sep='\t',index=False,header=False)
 print(test['preds'])
+
+# Calculate score
+score = Evaluation.calculate_score(test,"data/testdata/test_df_label.csv","data/testdata/test_df.csv","Results/LGBScore.txt")
+print(("#Your score is %.4f")%(score*100.0))
```

Comment:
So results won't be saved automatically?

Reply:
It is saved in results file. So what's your meaning of automatically?